docs/UserGuide.md

```diff
@@ -71,6 +71,22 @@ Examples:
 * `add c n/Apple p/98765432 e/applehr@example.com d/Top tech company t/tech t/interested`:
   ![add company with all information added](images/addCompany.png)
 
+### Adding an internship to a company contact: `add i`
+
+Adds an internship of interest to the corresponding company.
+
+Format: `add i n/ROLE_NAME d/DESCRIPTION [s/SCHEDULED_INTERVIEW_TIME]`
```

Comment:
Is this missing an `INDEX` parameter to determine what company the Internship is under?

Reply:
Thanks for catching this